[2025-06-23T21:08:49.753+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-23T21:08:49.790+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_training.cleanup_resources manual__2025-06-23T20:53:21.071224+00:00 [queued]>
[2025-06-23T21:08:49.805+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_training.cleanup_resources manual__2025-06-23T20:53:21.071224+00:00 [queued]>
[2025-06-23T21:08:49.807+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-06-23T21:08:49.828+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): cleanup_resources> on 2025-06-23 20:53:21.071224+00:00
[2025-06-23T21:08:49.839+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1491) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-23T21:08:49.842+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_training', 'cleanup_resources', 'manual__2025-06-23T20:53:21.071224+00:00', '--job-id', '239', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_training_dag.py', '--cfg-path', '/tmp/tmpbt2jp2f2']
[2025-06-23T21:08:49.843+0000] {standard_task_runner.py:72} INFO - Started process 1492 to run task
[2025-06-23T21:08:49.843+0000] {standard_task_runner.py:105} INFO - Job 239: Subtask cleanup_resources
[2025-06-23T21:08:49.909+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection_training.cleanup_resources manual__2025-06-23T20:53:21.071224+00:00 [running]> on host c79a0311c601
[2025-06-23T21:08:50.025+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Jasjot Parmar' AIRFLOW_CTX_DAG_ID='fraud_detection_training' AIRFLOW_CTX_TASK_ID='cleanup_resources' AIRFLOW_CTX_EXECUTION_DATE='2025-06-23T20:53:21.071224+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-23T20:53:21.071224+00:00'
[2025-06-23T21:08:50.027+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-06-23T21:08:50.056+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-06-23T21:08:50.059+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'rm -f /app/tmp/*.pkl']
[2025-06-23T21:08:50.080+0000] {subprocess.py:99} INFO - Output:
[2025-06-23T21:08:50.088+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-06-23T21:08:50.123+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-06-23T21:08:50.124+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fraud_detection_training, task_id=cleanup_resources, run_id=manual__2025-06-23T20:53:21.071224+00:00, execution_date=20250623T205321, start_date=20250623T210849, end_date=20250623T210850
[2025-06-23T21:08:50.188+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-23T21:08:50.210+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-23T21:08:50.213+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
